module-2/backend/week_two.md

```diff
@@ -13,8 +14,21 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
-
+maximum
+average
+group
+find_or_create_by
+where
+order
+uniq
+pluck
+joins
+=> through Activerecord. 
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+team = Team.where(id: 4)
+team.name 
+=> "Team Name"
+team.owner_id.pluck(:name)
```

Comment:
Can't use pluck here! `team.owner_id` is going to return a number (the id of the owner), and we can only call `pluck` on ActiveRecord objects. Instead we need to first find the owner itself.

```ruby
owner_4_id = Team.find(4).owner_id
Owner.find(owner_4_id)
```